category_encoders/one_hot.py

```diff
@@ -142,11 +142,21 @@ def fit(self, X, y=None, **kwargs):
             if X[self.cols].isnull().any().any():
                 raise ValueError('Columns to be encoded can not contain null')
 
+        # TODO: when these are settled, make the code sleeker
+        if self.handle_missing == 'error':
```

Comment:
maybe you can just use a dict here?
```
oe_handle_missing_mapping = {"error": "error", "return_nan": "return_nan", ... }
```
that way we'd also get a key error in case the option is invalid